.github/scripts/rebuild-build.sh

```diff
@@ -2,5 +2,8 @@
 set -ev
 ./gradlew --no-daemon --version
 ./gradlew --no-daemon -Dmaven.repo.local=dist/m2 :buildscriptDependencies :publish
-./gradlew --no-daemon -Dmaven.repo.local=dist/m2 --warning-mode=fail :gradle-plugins:build
+# Temporarily ignore warnings like "Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0."
```

Comment:
I don't think this is needed here. It is only on the rebuild-test execution that we are using JDK 24.